server/modules/instantiation/jobs.js

```diff
@@ -303,16 +305,19 @@ module.exports = function (config, portProvider, interpolationHelper, buildInsta
             var { buildInstance } = job;
```

Comment:
nah, I didn't literally copy it, tried the one without any typos and it didn't work :)